packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -2133,44 +2102,24 @@ module.exports = function(log, config, oauthdb) {
       serviceLastActiveDate,
     } = message;
 
-    if (!config.subscriptions.transactionalEmails.enabled) {
-      log.trace('mailer.subscriptionCancellation', {
-        enabled: false,
-        email,
-        productId,
-        uid,
-      });
-      return;
-    }
-
+    const enabled = config.subscriptions.transactionalEmails.enabled;
```

Comment:
Would we always need to have this config flag? Or is the plan to remove once its been live for a while?

Reply:
I wouldn't say there's a plan to, but it does make sense to remove this code once it's been live awhile yeah. We'll probably need to have it live on stage but not prod for a good bit, while we QA and coordinate with IT to turn off their version of all these emails